server.js

```diff
@@ -1,30 +1,109 @@
 import express from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Defines the port the app will run on, can be overridden 
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+// Middlewares to enable cors and json body parsing (should not be changed)
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+
+// ROUTES
+
+//Startpage with definition of routes
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  
+  res.send({
+    "Books API" : "Look for books by ID, ISBN number or title",
+    "Routes":[{
+      "/booksData": "Get all books",
+      "/booksData/isbn/:isbn": "Get a book by its ISBN number",
+      "/booksData/id/:id": "Get a book by its ID number in the database",
+      "/booksData/authors/:authors": "Get a book by its author/authors",
+    }]
+  });
```

Comment:
I like the fact that you have a small "table of content" so it's a lot easier to find your routes! Nice :)